src/io/read_vtk.jl

```diff
@@ -31,18 +36,22 @@ ic = vtk2trixi(joinpath("out", "rectangular.vtu"))
 │ eltype: …………………………………………………………… Float64                                                          │
 │ coordinate eltype: ……………………………… Float64                                                          │
 └──────────────────────────────────────────────────────────────────────────────────────────────────┘
-````
+```
 """
-function vtk2trixi(file)
+function vtk2trixi(file; element_type=:default, coordinates_eltype=Float64)
```

Comment:
```suggestion
function vtk2trixi(file; element_type=nothing, coordinates_eltype=Float64)
```
Is there a reason why you need to use a "magic value" symbol instead of just `nothing`? We always preferred `nothing` in these cases.